test/test_mpo.jl

```diff
@@ -52,6 +52,21 @@ using ITensors,
     badpsi = randomMPS(badsites)
     @test_throws DimensionMismatch applyMPO(K,badpsi)
   end
+
+  @testset "nmultMPO" begin
+    psi = randomMPS(sites)
+    K = randomMPO(sites)
+    L = randomMPO(sites)
+    @test maxDim(K) == 1
+    @test maxDim(L) == 1
```

Comment:
Later would be good to test nmult for MPOs with > 1 bond dimension. We can use AutoMPO for example to get some non-trivial MPOs